src/plugins/gs-plugin-appstream.c

```diff
@@ -745,8 +745,6 @@ gs_plugin_add_categories (GsPlugin *plugin,
 		app = g_ptr_array_index (array, i);
 		if (as_app_get_id (app) == NULL)
 			continue;
-		if (as_app_get_priority (app) < 0)
```

Comment:
Is this change alone not sufficient for the core apps to load in the categories?
Would this not be needed if we had appstream data for the core apps?